src/features/events/components/EventParticipantsTable.tsx

```
@@ -0,0 +1,262 @@
+import Text from '@/components/base/Text'
+import ProfilePicture from '@/components/ProfilePicture'
+import SkeCard from '@/components/Skeleton/CardSkeleton'
+import { useSuspensePaginatedEventPartcipants } from '@/services/events/hook'
+import { getHumanFormattedDate } from '@/utils/date'
+import { useMedia, XStack, YStack } from 'tamagui'
+
+const EndTable = (props: { children: ReactNode }) => {
+  const media = useMedia()
+
+  return media.sm || media.gtLg ? (
+    <Table.ScrollView splited="end" flex={1} children={props.children} horizontal contentContainerStyle={{ flexGrow: 1 }} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)